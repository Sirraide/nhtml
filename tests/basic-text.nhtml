div {
    div { text (text1) }
    p { text (text2) }
}