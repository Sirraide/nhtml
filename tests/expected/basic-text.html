<div>
    <div>text1</div>
    <p>text2</p>
</div>